hazelcast/src/main/resources/hazelcast-config-3.10.xsd

```diff
@@ -1323,6 +1324,21 @@
             </xs:simpleType>
         </xs:attribute>
     </xs:complexType>
+    <xs:complexType name="count-down-latch">
+        <xs:all>
+            <xs:element name="quorum-ref" minOccurs="0" maxOccurs="1"/>
```

Comment:
Shouldn't we specify a type like `type="xs:string"` here? We didn't do this for any `quorum-ref` so far, but I'm not sure what the default type is.

Reply:
I just followed the other quorum-ref definitions. Don't want to change it to a different type in one part and have it done differently in the older parts. Could be cleaned up in 4.0 IMHO.